tests/client/core/api/test_api.js

```diff
@@ -1,24 +1,33 @@
-/* global window */
+/* global Response, window */
 import config from 'config';
+import { oneLine } from 'common-tags';
 
 import * as api from 'core/api';
 import { ADDON_TYPE_THEME } from 'core/constants';
 import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+export function generateHeaders(
+  headerData = { 'Content-Type': 'application/json' }
+) {
+  const response = new Response();
+  Object.keys(headerData).forEach((key) => (
+    response.headers.append(key, headerData[key])
+  ));
+  return response.headers;
+}
+
 function createApiResponse({
   ok = true, jsonData = {}, ...responseProps
 } = {}) {
   const response = {
```

Comment:
I think it would be wise to use a real response. Can you file an issue so we don't forget?